src/main/java/com/parkhomovsky/bookstore/exception/GlobalExceptionHandler.java

```diff
@@ -88,6 +91,47 @@ protected ResponseEntity<Object> handleJwtException(JwtException ex) {
         return new ResponseEntity<>(body, HttpStatus.FORBIDDEN);
     }
 
+    @ExceptionHandler(UserNotAuthenticatedException.class)
+    protected ResponseEntity<Object> handleJwtException(UserNotAuthenticatedException ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
+        body.put("status", HttpStatus.FORBIDDEN);
+        body.put("error", ex.getMessage());
+        return new ResponseEntity<>(body, HttpStatus.FORBIDDEN);
+    }
+
+    @ExceptionHandler(InvalidRequestParametersException.class)
+    public ResponseEntity<Object> handleInvalidRequestParametersException(
+            InvalidRequestParametersException ex
+    ) {
+        Throwable rootCause = ExceptionUtils.getRootCause(ex);
+        String errorMessage = "An error occurred while proceeding request data.";
+        if (rootCause != null) {
+            errorMessage = "Error: Wrong request data. " + rootCause.getMessage();
+        }
+        Map<String, Object> errorMessageBody = new LinkedHashMap<>();
+        errorMessageBody.put("timestamp", LocalDateTime.now());
+        errorMessageBody.put("status", HttpStatus.FORBIDDEN);
```

Comment:
Are you sure that HttpStatus.FORBIDDEN is the best option in case of InvalidRequestParametersException?